### DISCUSSION TITLE
Does Plausible work with MKDoc

### DISCUSSION THREAD

**Discussion by russurquhart** [plausible/analytics#2335] [Self-Hosted Support]
Hi,
I'm doing research for moving our doc site potentially to MKDoc. Can Plausible work with MKDoc? Has anyone used it as a plugin?

Thanks,

Russ

**Comment by ruslandoga** [comment]
👋

You probably can add a script tag with tracker to your root template, it'd depend on your theme. Plugins are unlikely to be useful :)
